### DISCUSSION TITLE
v2: 请问Field的内置校验成功后才执行异步校验，这样的场景怎样实现？

### DISCUSSION THREAD

**Discussion by effapp** [alibaba/formily#1273] [Q&A 2.x] [answered]
v2: 请问Field的内置校验成功后才执行异步校验，这样的场景怎样实现？

例如：注册场景中，我想手机号码合法化校验后，继续进行异步校验这个号码是否已经注册过，怎样实现这样的功能？

**Comment by janryWang** [comment, marked as answer]
validator={[{required:true},"phone",()=>这里写异步校验逻辑]} 按照顺序来就行
